crypto/transactions/serializers/htlc_claim.py

```diff
@@ -9,7 +10,10 @@ class HtlcClaimSerializer(BaseSerializer):
 
     def serialize(self):
         self.bytes_data += unhexlify(self.transaction['asset']['claim']['lockTransactionId'])
-        unlock_secret = hexlify(self.transaction['asset']['claim']['unlockSecret'].encode())
```

Comment:
this was an unnecessary hexlify/unhexlify that ends up results in the same thing